controller/iteration_test.go

```diff
@@ -86,6 +96,17 @@ func (rest *TestIterationREST) TestSuccessCreateChildIteration() {
 	require.NotNil(rest.T(), created.Data.Relationships.Workitems.Meta)
 	assert.Equal(rest.T(), 0, created.Data.Relationships.Workitems.Meta["total"])
 	assert.Equal(rest.T(), 0, created.Data.Relationships.Workitems.Meta["closed"])
+
+	// try to create child iteration with some other user
+	otherIdentity := &account.Identity{
+		ID:           uuid.NewV4(),
```

Comment:
Thanks, yes no need to set ID explicitly.
done in https://github.com/almighty/almighty-core/pull/1347/commits/dd3159268e68f2e0036df69c31f208e104325781